packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/RemoteMessageSerializer.java

```diff
@@ -6,6 +6,7 @@
 
 import java.util.Map;
 
+import androidx.annotation.NonNull;
```

Comment:
Should we use our own here?

```suggestion
import expo.modules.core.interfaces.DoNotStrip;
```